src/LoaderManager.php

```diff
@@ -101,4 +110,24 @@ private function registerDefaultWebLoaders()
             $this->loaders['http']  = new FileGetContentsWebLoader('http://');
         }
     }
+
+    /**
+     * @return DecoderManager
+     */
+    public function getDecoderManager()
```

Comment:
Why does the LoaderManager need to know about the DecoderManager?  It shouldn't need to know about the dependencies of it's dependencies.

Reply:
I want to be able to extend the supported formats like this:

$deref  = new League\JsonReference\Dereferencer();
$deref->getLoaderManager()->getDecoderManager()->registerDecoder('xml', $customDecoder);